tst/EnergyPlus/unit/PurchasedAirManager.unit.cc

```diff
@@ -149,23 +149,20 @@ TEST_F(EnergyPlusFixture, SizePurchasedAirTest_Test1)
     DataEnvironment::StdRhoAir = 1.0; // Prevent divide by zero in ReportSizingManager
     ZoneEqSizing(CurZoneEqNum).SizingMethod.allocate(24);
     CurSysNum = 0;
-    ZoneHVACSizing.allocate(1);
-    ZoneHVACSizing(1).CoolingSAFMethod = SupplyAirFlowRate;
-    ZoneHVACSizing(1).HeatingSAFMethod = SupplyAirFlowRate;
 
-    ZoneEqSizing(CurZoneEqNum).AirVolFlow = 0.0;
     FinalZoneSizing.allocate(1);
+    FinalZoneSizing(CurZoneEqNum).MinOA = 0.0;
+    FinalZoneSizing(CurZoneEqNum).OutTempAtHeatPeak = 5.0;
     FinalZoneSizing(CurZoneEqNum).DesHeatVolFlow = 1.0;
-    ZoneEqSizing(CurZoneEqNum).HeatingAirVolFlow = 1.0;
     FinalZoneSizing(CurZoneEqNum).DesHeatCoilInTemp = 30.0;
```

Comment:
The unit test shouldn't be setting ZoneEqSizing - the component sizing function should be doing that.